src/visitor.cpp

```diff
@@ -237,6 +237,13 @@ bool chimera::Visitor::GenerateGlobalVar(clang::VarDecl *decl)
         return false;
     else if (!decl->isThisDeclarationADefinition())
         return false;
+    // Ignore unspecialized template variables.
+    else if (decl->getDescribedVarTemplate())
+        return false;
+    // TODO(#297): Ignore specialized template variables because the current
+    // variable mastch and template not able to generate valid binding.
```

Comment:
What is `mastch`?

Reply:
I meant `mstch` or `mustache` trying to refer to the `::match::object`. What would be a better notion?

Reply:
Oh ok, then `mstch` is fine, it is just a typo.